funding/manager.go

```diff
@@ -3391,6 +3565,35 @@ func (f *Manager) waitForZeroConfChannel(c *channeldb.OpenChannel,
 	return nil
 }
 
+// genFirstStateMusigNonce generates a nonces for the "first" local state. This
+// is the verification nonce for the state created for us after the initial
+// commitment transaction signed as part of the funding flow.
+func genFirstStateMusigNonce(channel *channeldb.OpenChannel,
```

Comment:
looks more generic than "first state" since it uses `LocalCommitmennt.CommitHeight+1` as input.

Reply:
True, thought at this point, the channel can't have progressed past hte first state as funding locked hasnt' been sent, and only later do we actually send it to the switch.

Reply:
It can be if this is the scid-alias upgrade case, but in that case we correctly ignore the nonces past the first state